motr/st/utils/be_log_set.sh

```
@@ -0,0 +1,88 @@
+#!/bin/basih
+
+# Script to set BE log size
+# 1.parse hctl status output
+# 2. list size of existing services log file
+# 3. set new size using m0betool
+# 4. list size of all io services log file
+
+M0D_LIST_FID=""
+BE_LOG_SIZE=268435456
+
+function usage()
+{
+   echo "be_log_set.sh [-s <be_log_size>]"
+   echo "Options:"
+   echo "    '-s <size>'  BE log size to change"
+   echo "Example.."
+   echo "./be_log_set.sh"
+   echo "./tune_addb.sh -s <BE log size to change>"
+   exit 1
+}
+
+change_be_log_size() {
+    for fid in $M0D_LIST_FID; do
+    	local log_file="/var/motr/m0d-${fid}/db-log/o/0:28"
+    	local db_file="/var/motr/m0d-${fid}/db"
+	echo "Before setting BE log size log_file=$log_file"
+	ls -lsh $log_file
+	echo "db_file=$db_file"
+	echo "m0betool create_be_log $db_file $BE_LOG_SIZE"
+	m0betool create_be_log $db_file $BE_LOG_SIZE
+	[[ $? -ne 0 ]] && echo "Failed" && exit 1
+	echo "After setting BE log size log_file=$log_file"
+	ls -lsh $log_file
+    done
+}
+
+_local_params() {
+    local addr="([0-9]+[.][0-9]+[.][0-9]+[.][0-9]+)"
+    local host
+    local M0D_LIST_NET_ADDR
+
+    if [[ $(cat /var/lib/hare/node-name) == "srvnode-1" ]]; then
+	host="srvnode-1"
+    elif [[ $(cat /var/lib/hare/node-name) == "srvnode-2" ]]; then
+	host="srvnode-2"
+    else
+	exit 1
```

Don't silently fail and exit.
Please print/log error numbers and messages.